packages/platform/platform-fastify/src/components/PlatformFastify.ts

```
@@ -0,0 +1,337 @@
+  Platform,
+  PlatformAdapter,
+  PlatformApplication,
+  PlatformBuilder,
+  PlatformMulterSettings,
+  PlatformRequest,
+  PlatformResponse,
+  PlatformStaticsOptions
+} from "@tsed/platform-http";
+import {PlatformHandlerMetadata, PlatformHandlerType, PlatformLayer} from "@tsed/platform-router";
+import Fastify, {FastifyInstance, FastifyReply, FastifyRequest} from "fastify";
+import {IncomingMessage, ServerResponse} from "http";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)